mono/metadata/sgen-tarjan-bridge.c

```diff
@@ -124,7 +130,26 @@ typedef struct _ScanData {
 	unsigned obj_state : 2;
 } ScanData;
 
+#define HEAVY_REFS_MIN 2
+#define HEAVY_COMBINED_REFS_MIN 60
```

Comment:
It would be nice to move those knobs and the one before (INCOMING_COLORS_BITS) together and include a description of how you picked the values and what's the impact in changing them.

Model it after this: https://github.com/mono/mono/blob/master/mono/sgen/sgen-conf.h